src/Menu.js

```diff
@@ -51,7 +51,7 @@ class Menu extends React.Component {
 
   // Start menu animation
   _onMenuLayout = (e) => {
-    if (this.state.menuState === STATES.ANIMATING) {
+    if ([STATES.HIDDEN, STATES.ANIMATING].includes(this.state.menuState)) {
```

Comment:
Why don't we just check for the remaining `states` option then?
```suggestion
    if (this.state.menuState !== STATES.SHOWN) {
```

Reply:
Yes! I forgot there were only 3 states haha